schemas/items_schema.json

```diff
@@ -12,6 +12,10 @@
         "type": "integer",
         "minimum": 0
       },
+      "textId": {
```

Comment:
isn't that the same as name ?

Reply:
It's probably the same that's happening with the block schema

Reply:
Yes